packages/fast-components-react-base/src/breadcrumb/examples.data.tsx

```diff
@@ -19,10 +19,6 @@ const label: any = {
     label: "breadcrumb",
 };
 
-function renderSeparator(): (className?: string) => React.ReactNode {
-    return (className?: string): React.ReactNode => <div className={className}>\</div>;

```

Comment:
Maybe just change the return value of this function instead of duplicating the object?